performance_suites/benchmark.py

```diff
@@ -0,0 +1,137 @@
+import timeit
```

Comment:
We should use https://pyperf.readthedocs.io/ that produces way more stable and accurate results.